src/pages/Contributing/GitConventions/index.md

```diff
@@ -28,49 +26,47 @@ Each commit message consists of a `type`, an optional `scope` and a `description
 feat(component/button): add a warning color theme
 ```
 
-> Commit message's lines **cannot be longer than 140 characters**! <br> This leads to **more readable messages** that are easy to follow when looking through the project history. 
+> Commit message's lines **cannot be longer than 140 characters**! <br> This leads to **more readable messages** that are easy to follow when looking through the project history.
 
-We are **automating the releases version numbers (SEMVER) and the generation of the changelogs files by parsing the commit messages**. 
+We are **automating the releases version numbers (SEMVER) and the generation of the changelogs files by parsing the commit messages**.
 
 That's why it's very important to follow the conventional commit conventions, and to be very thoughtful about the description you provide
```

Comment:
link for conventional commit conventions? 

and to be very thoughtful -> and to be as clear as possible on the description